problem_builder/dashboard.py

```diff
@@ -172,6 +172,21 @@ class DashboardBlock(StudioEditableXBlockMixin, XBlock):
         ).format(example_here='["2754b8afc03a439693b9887b6f1d9e36", "215028f7df3d4c68b14fb5fea4da7053"]'),
         scope=Scope.settings,
     )
+    exclude_questions = Dict(
+        display_name=_("Questions to be hidden"),
+        help=_(
+            "Optional rules to exclude specific questions both from displaying in dashboard and from average "
+            "calculation. Rules must start with url_name of mentoring block, followed by list of question numbers "
+            "to exclude. Rule set must be in JSON format. Question numbers are one-based (i.e. first question "
+            "has number 1)"
```

Comment:
Also, I'd suggest you rephrase: "Optional rules to exclude specific questions both from displaying in dashboard and from **the** calculated average. Rules must start with **the** url_name of **a** mentoring block, followed by list of question numbers to exclude (**the first question being number 1**). **Must be in JSON format. Examples:** ..."